modules/ztp-installation-crs.adoc

```diff
@@ -22,7 +22,7 @@ The following table lists the installation CRs that are automatically applied by
 
 |`BareMetalHost`
 |Contains the connection information for the Baseboard Management Controller (BMC) of the target bare-metal host.
-|Provides access to the BMC to load and boot the discovery image on the target server by using the Redfish protocol. ZTP supports iPXE and virtual media network booting.
+|Provides access to the BMC to load and boot the discovery image on the target server by using the Redfish protocol.
```

Comment:
```suggestion
|Provides access to the BMC to load and start the discovery image on the target server by using the Redfish protocol.
```

Reply:
The IBM Style Guide advises that we use "start" or "turn on" instead of "boot" where possible.